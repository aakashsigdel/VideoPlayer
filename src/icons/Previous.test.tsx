import * as React from 'react'
import { create } from 'react-test-renderer'

import { Previous } from './Previous'

describe('<Previous />', () => {
  it('should render correctly', () => {
    const tree = create(<Previous />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
