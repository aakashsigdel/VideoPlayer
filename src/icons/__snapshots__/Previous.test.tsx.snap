// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Previous /> should render correctly 1`] = `
<svg
  fill="#fff"
  height="24"
  viewBox="0 0 24 24"
  width="24"
>
  <path
    d="M6 6h2v12H6zm3.5 6l8.5 6V6z"
  />
  <path
    d="M0 0h24v24H0z"
    fill="none"
  />
</svg>
`;
